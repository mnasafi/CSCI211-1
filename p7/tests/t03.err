insert <Hello world.> failed. String already in tree.
