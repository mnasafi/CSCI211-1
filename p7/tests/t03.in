echo t03: Insert "Hello world." into tree twice.
insert Hello world.
echo Inserting "Hello world." the second time, should print error.
insert Hello world.
