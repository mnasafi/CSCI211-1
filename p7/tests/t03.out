t03: Insert "Hello world." into tree twice.
Inserting "Hello world." the second time, should print error.
